### PR TITLE
Feature (storages): Add Ephemeral JSON Storage.

### PR DESCRIPTION
**Pull request by imalento**
Hi! I have made tinydb work on GoogleAppEngine.
Please try using it.

- Open JSON read only
- Write on memory

### CODE REVIEW

File: tinydb/storages.py [hunk 1]
```diff
@@ -134,3 +134,44 @@ def read(self):
 
     def write(self, data):
         self.memory = data
+
+
+class EphemeralJSONStorage(Storage):
```
Comment (eugene-eeo) [inline tinydb/storages.py]: IMO you should subclass JSONStorage instead, that way you just need to override `write` and on a `read` you can do something like:

```
def read(self):
    if self.memory is not None:
        return self.memory
    return super().read(self)
```

**Comment by msiemens** [comment]
Hey @imalento, thanks for the pull request :) What do you think about making this an extension, something like `tinydb-appengine`? If you make a small repository for this, I'll add it to the list of extensions :)

**Comment by imalento** [comment]
Hi! Thank you for checking!
I also thought that this should be extension.
I'll make another small repository :smile:

**Comment by msiemens** [comment]
Awesome, thanks 🙂

**Comment by msiemens** [comment]
Just comment here when the repo is ready to be included in the extensions list 🙃

File: tinydb/storages.py (hunk 1 above)
Comment (imalento) [reply tinydb/storages.py]: Thanks! I should have done so.

**Comment by imalento** [comment]
Hi, I have made a repository. Thanks so much!
https://github.com/imalento/tinydb-appengine

**Comment by msiemens** [comment]
Thanks, it's now in the list of extensions: https://tinydb.readthedocs.io/en/latest/extensions.html#tinydb-appengine 🙂

**Pull request closed without merging**